server/views/applications/before-you-start.njk

```diff
@@ -29,32 +31,30 @@
     <div class="govuk-grid-column-two-thirds">
       <h2>Before you start</h2>
 
-      <h3>Who is this for?</h3>
+      <h3>You need:</h3>
 
       <ul>
-        <li>people without suitable accommodation in the community</li>
-        <li>people on Bail Order</li>
-        <li>people on a Home Detention Curfew (HDC) licence </li>
-        <li>people referred who are homeless on conditional release (HCRD)</li>
-        <li>people on licence considered for recall (ATCR)</li>
-      </ul>
-
-      <h3>You need</h3>
-
-      <ul>
-        <li>to have written consent from the applicant to apply for CAS-2 accommodation</li>
+        <li>consent from the applicant</li>
         <li>the applicant’s CRN</li>
-        <li>an up to date OASys – we'll use this to pre-fill some of your application</li>
-        <li>factors that will have an impact on the placement location</li>
+        <li>the applicant's prison number (from NOMIS)</li>
+        <li>a Layer 3 OASys for the applicant, which has been updated in the last 6 months</li>
       </ul>
 
-      <h4>If you need help using this service</h4>
-
-      <p>If you have any questions about the referral process or application forms, you can call the CAS-2 Referral Hub on <a href="tel:0300 555 0264">0300 555 0264</a> or <a href="tel:0300 555 0264">07423 434032</a> (alternative number) at the following times:</p>
-
-      <p>Monday to Friday, 8am-6pm </br> Saturday, 8am-1pm</p>
+      <details class="govuk-details">
+        <summary class="govuk-details__summary">
+          <span class="govuk-details__summary-text">
+            What to do if OASys is not up to date
+          </span>
+        </summary>
+        <div class="govuk-details__text">
+          <p class="govuk-body-text">Information in the application should be up to date and consistent with OASys.</p>
```

Comment:
if you put the `govuk-body-text` class on the parent div, don't need to declare it on the `<p>` tags within 😎

Reply:
Thanks for letting me know, I think I've removed it now? 🤞

Reply:
nice one - if you add it to the div now, it will be inherited by all the <p> tags. 

i.e. `<div class="govuk-details__text govuk-body-text">`